react-components/src/components/Image360Details/Image360Details.tsx

```diff
@@ -9,7 +9,11 @@ import { useReveal } from '../..';
 import { type Image360 } from '@cognite/reveal';
 import { Button } from '@cognite/cogs.js';
 
-export function Image360Details(): ReactElement {
+type Image360DetailsProps = {
+  appLanguage?: string;
+};
+
+export function Image360Details({ appLanguage }: Image360DetailsProps): ReactElement {
```

Comment:
How will this work for external users? Will they be able to find the active language e.g. through cdf-utils i18n wrapper?

Reply:
In `cdf-utils` they have exposed a method `getLanguage()` for active language and for any other app, they should support it if not browser supported language will be chosen